### ISSUE TITLE
[Question] Is there any way to externally add recipes for the combiner?

### ISSUE THREAD

**Issue by Ragnaraven** [al132mc/alchemistry#329]
We can dissolve Glowstone and Quartz, but there is no way to remake them.

Is there any way to add recipes in a datapack or something like it to enable combining for these and items that are dissolvable but not creatable?

**Comment by Geolykt** [comment]
See <https://github.com/SmashingMods/Alchemistry/tree/1.20.1/src/generated/resources/data/alchemistry/recipes/combiner>

**Comment by Ragnaraven** [comment]
@Geolykt Forgive me, but I actually am there already! Is there a standard way to add recipes to a mod without branching it and sticking them in there (which is what I am currently doing)?

**Comment by Geolykt** [comment]
Yes, via Datapacks. KubeJS might also be capable of that but I have even less experience (read: Nada) with it.

**Comment by Ragnaraven** [comment]
Thank you. I tried a datapack before I asked here because I had no success with it. I will see if I can do it and post an example here if I can get one to work for anyone's future reference.

**Comment by Ragnaraven** [comment]
[ragnaraven_alchem_pack.zip](https://github.com/SmashingMods/Alchemistry/files/12824256/ragnaraven_alchem_pack.zip)

@Geolykt Would you be able to review this pack for me please? Its really simple. The pack is loaded successfully (/datapacks list has it present). I am using 1.18.2, it is in a single player world/datapacks folder. The recipes do not show up in JEI or work in the combiner. Not sure what I am doing wrong.

**Comment by Geolykt** [comment]
It could be that you don't need the "alchemistry" folder. I'd take a look how you add vanilla recipes and follow that

**Comment by kienerxyzzy** [comment]
I also tried it once (#352).
I tried datapacks. Failed.
I tried injecting into an existing datapack. Failed, and broken the world with the datapack.
I tried KubeJS. Failed.
I tried un- and re- zipping the .jar file. Failed, and also remove it from the mod list entirely.
What should I do and what am I doing wrong?
